core/src/confidence.rs

```diff
@@ -52,6 +52,33 @@ impl ForkConfidenceCache {
     pub fn total_stake(&self) -> u64 {
         self.total_stake
     }
+
+    pub fn get_fork_with_depth_confidence(
+        &self,
+        minimum_depth: u64,
```

Comment:
nit: usize to avoid the `as usize` typecast below, and MAX_LOCKOUT_HISTORY is also defined as usize?